src/test/kotlin/autolotto/AutoLottoTest.kt

```diff
@@ -0,0 +1,44 @@
+package autolotto
+
+import autolotto.vo.AutoLotto
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class AutoLottoTest {
+    @Test
+    fun `로또 구입금액을 입력받으면 로또 수량이 결정된다`() {
+        // Given
+        val input = 1000L
+
+        // When
+        val lotto = AutoLotto(input)
+
+        // Then
+        lotto.count shouldBe 1L
+    }
+
+    @Test
+    fun `로또 구입금액 1500원을 입력받으면 로또 수량은 1개로 결정된다`() {
+        // Given
+        val input = 1500L
+
+        // When
+        val lotto = AutoLotto(input)
+
+        // Then
+        lotto.count shouldBe 1L
+    }
+
+    @Test
+    fun `구매수량만큼 로또를 발급한다`() {
+        // Given
+        val input = 10000L
+
+        // When
+        val lotto = AutoLotto(input)
+
+        // Then
+        lotto.count shouldBe 10L
+        lotto.lottos.size shouldBe 10
+    }
+}
```

Comment:
```suggestion
class AutoLottoTest: FunSpec ({

    test("로또 구입금액을 입력받으면 로또 수량이 결정된다") {
        // Given
        val input = 1000L

        // When
        val lotto = AutoLotto(input)

        // Then
        lotto.count shouldBe 1L
    }

    test("로또 구입금액 1500원을 입력받으면 로또 수량은 1개로 결정된다") {
        // Given
        val input = 1500L

        // When
        val lotto = AutoLotto(input)

        // Then
        lotto.count shouldBe 1L
    }

    test("구매수량만큼 로또를 발급한다") {
        // Given
        val input = 10000L

        // When
        val lotto = AutoLotto(input)

        // Then
        lotto.count shouldBe 10L
        lotto.lottos.size shouldBe 10
    }
})
```

p3; 이미 shouldBe와 같은 kotest 메서드를 활용해 주고 계신 것으로 보여, 간단하게 kotest로 마이그레이션할 수 있는 FunSpec부터 차근차근 시작해 보시는 건 어떨까요?

테스트가 실패한다면, kotest 버전을 올리시면 됩니다. 😊
testImplementation("io.kotest", "kotest-runner-junit5", "5.5.1")

Reply:
넵 ^.^ 담 PR에는 kotest로 적용해서 올리겠습니당!